dev/auth/authCtrl.js

```diff
@@ -19,6 +19,16 @@
       });
     }
 
+    vm.register = function(email, password, confirmPassword) {
+      authService.register(email, password, confirmPassword).then(function (res) {
+        if (res.data.error) {
```

Comment:
dev/interseptor.js
in 'responseError'
'return response;' should be 'return $q.reject(response);'